src/Compilers/Core/Portable/StrongName/StrongNameKeys.cs

```diff
@@ -95,7 +95,6 @@ internal static StrongNameKeys Create(string keyFilePath, CommonMessageProvider
 
             try
             {
-                Debug.Assert(PathUtilities.IsAbsolute(keyFilePath));
```

Comment:
Here I am, the user, passing  relative path on cmd line. You can either reject it or pass to the file IO transparently. We currently do the second.
Regardless, the assert seems not actionable.